documentation/markup.rst

```diff
@@ -565,6 +565,16 @@ The directives are:
 
             Description of the attribute.
 
+   This directive should be used for defining attributes.::
+
+      .. attribute:: my-attr
+         This attribute `my-attr` should be defined with this directive.
+
```

Comment:
We can delete this bit, it's essentially duplicating the "ham" example just before:
```suggestion
```